mlflow/models/utils.py

```diff
@@ -301,6 +302,9 @@ def __init__(self, input_example: ModelInputExample):
             self.info[EXAMPLE_PARAMS_KEY] = "true"
         model_input = deepcopy(self._inference_data)
 
+        if isinstance(model_input, pydantic.BaseModel):
+            model_input = model_input.model_dump()
```

Comment:
Are we ok with not supporting Pydantic v1 support here?

Reply:
Good catch :) I forgot to add this